app/src/main/java/com/duckduckgo/app/email/EmailJavascriptInterface.kt

```diff
@@ -17,35 +17,40 @@
 package com.duckduckgo.app.email
 
 import android.webkit.JavascriptInterface
-import timber.log.Timber
+import android.webkit.WebView
+import com.duckduckgo.app.browser.DuckDuckGoUrlDetector
+import com.duckduckgo.app.global.DispatcherProvider
+import kotlinx.coroutines.runBlocking
 
 class EmailJavascriptInterface(
     private val emailManager: EmailManager,
-    private val showNativeTooltip: () -> Unit
+    private val showNativeTooltip: () -> Unit,
```

Comment:
nit: Maybe not applicable to this class and how it's used but I would always generally place the lamda param last. That allows for kotlin niceties like define the labda after the `{`

Reply:
👍